code/modules/mob/living/carbon/human/human_examine.dm

```diff
@@ -219,19 +195,48 @@
 					msg += "<span class='deptradio'>Employment records: [copytext_preserve_html(skills, 1, char_limit-3)]...</span><a href='byond://?src=[UID()];employment_more=1'>More...</a>\n"
 
 
-	if(hasHUD(user,EXAMINE_HUD_MEDICAL))
+	if(hasHUD(user, EXAMINE_HUD_MEDICAL_READ))
 		var/perpname = get_visible_name(TRUE)
 		var/medical = "None"
+		var/mental = "None"
 
 		for(var/datum/data/record/E in GLOB.data_core.general)
 			if(E.fields["name"] == perpname)
 				for(var/datum/data/record/R in GLOB.data_core.general)
 					if(R.fields["id"] == E.fields["id"])
 						medical = R.fields["p_stat"]
+						mental = R.fields["m_stat"]
 
-		msg += "<span class = 'deptradio'>Physical status:</span> <a href='?src=[UID()];medical=1'>\[[medical]\]</a>\n"
+		var/medical_status = hasHUD(user, EXAMINE_HUD_MEDICAL_WRITE) ? "<a href='?src=[UID()];medical=1'>\[[medical]\]</a>" : "\[[medical]\]"
+		var/mental_status = hasHUD(user, EXAMINE_HUD_MEDICAL_WRITE) ? "<a href='?src=[UID()];mental=1'>\[[mental]\]</a>" : "\[[mental]\]"
+		msg += "<span class = 'deptradio'>Physical status: </span>[medical_status]\n"
+		msg += "<span class = 'deptradio'>Mental Status: </span>[mental_status]\n"
 		msg += "<span class = 'deptradio'>Medical records:</span> <a href='?src=[UID()];medrecord=`'>\[View\]</a> <a href='?src=[UID()];medrecordadd=`'>\[Add comment\]</a>\n"
```

Comment:
```suggestion
		msg += "<span class='deptradio'>Physical status: </span>[medical_status]\n"
		msg += "<span class='deptradio'>Mental Status: </span>[mental_status]\n"
		msg += "<span class='deptradio'>Medical records:</span> <a href='?src=[UID()];medrecord=`'>\[View\]</a> <a href='?src=[UID()];medrecordadd=`'>\[Add comment\]</a>\n"
```
I know this is part of the copy paste, but can you fix this class attribute spacing while we are here?